File: Uva/UVa10222.py
string = input().lower()

# keyboard = '234567890-=ertyuiop[]dfghjkl;\'cvbnm,./'
# translate = '`1234567890qwertyuiopasdfghjklzxcvbnm,'

table = '`1234567890-=qwertyuiop[]\\asdfghjkl;\'zxcvbnm,./'
# print(keyboard.index('e'))

for i in string:
    if i == ' ':
        print(end=' ')
    else:
        # print(i)
        index = table.index(i)
        print(table[index-2], end='')

#??
print('')

File: UVa_cpp/UVa10222.cpp
#include <iostream>

using namespace std;

string str = "qwertyuiop[]\\asdfghjkl;'zxcvbnm,./";

int main(){
    int n;
    string s;
    // cin >> n;
    // getline(cin, s);
    while(getline(cin, s)){
        // getline(cin, s);
        for(int i=0; i<s.size(); i++){
            if ('A' <= s[i] && s[i] <= 'Z') s[i] += 'a' - 'A';
            // cout << (char)s[i] << '\n';
            if (s[i] == ' '){
                cout << ' ';
                continue;
            }
            for(int j=0; j<str.size(); j++){
                if ((char)s[i] == str[j])
                    cout << (str[j-2]);
            }
        }
        cout << '\n';
    }
}